### PR TITLE
Proof P11 #230

### PR DESCRIPTION
**Pull request by dwiyatci**
🕵🏻 Proofread #230.

### CODE REVIEW

File: Comanche055/P11.agc [hunk 1]
```diff
@@ -858,7 +864,7 @@ SATSTKON	EXTEND
 		RELINT
 		TC	GOPIN		# EXIT THUS BECAUSE WE CAME VIA V46
 
-		SBANK=	PINSUPER	# Added RSB 2009
```
Comment (dwiyatci) [inline Comanche055/P11.agc]: This line is actually found nowhere in the original manuscript, but I guess it's required for the program to work correctly, thus I didn't remove it and improve its comment.

**Comment by wopian** [comment]
I'll be going through all of the open PRs throughout this week 👍 (got half way through this one before I was held off by other things last week)

**Comment by dwiyatci** [comment]
> I'll be going through all of the open PRs throughout this week 👍 (got half way through this one before I was held off by other things last week)

Take your time James, I'm also taking mine. 🙏

File: Comanche055/P11.agc (hunk 1 above)
Comment (dwiyatci) [reply Comanche055/P11.agc]: Ok now, I believe based on @oldmud0's comment on the other PR, I have to remove this line as well.

**Pull request merged**